test/comps/test_util.cc

```diff
@@ -28,7 +28,50 @@ class TestUtil : public ::testing::Test
 
     template <typename T>
     static void 
-    test_1()
-    {
+    test_2_norm(int64_t m, int64_t n, std::tuple<int, T, bool> mat_type, RandBLAS::base::RNGState<r123::Philox4x32> state) {
```

Comment:
``test_matrix_2_norm`` or ``test_spectral_norm``

Reply:
Please rename the utility function itself to ``matrix_2_norm`` or ``spectral_norm``.

Reply:
Or maybe ``estimate_matrix_2_norm`` or ``estimate_spectral_norm``

Reply:
Done. went with estimate_spectral_norm.